mod.ts

```diff
@@ -187,6 +217,34 @@ export type ColorFormat = Readonly<{
   accent: boolean;
 }>;
 
+export type AnsiColorGroups = Readonly<{
+  /**
+   * An object containing all the ANSI "normal" colors, which are the 8 standard colors from 0 to 7.
+   */
+  normal: AnsiColorFormat;
+
+  /**
+   * An object containing all the ANSI "bright" colors, which are the 8 standard colors from 8 to 15.
+   * 
+   * Note: These bright colors are not neccesarily "brighter" than the normal colors, but rather more bold and saturated.
+   */
+  bright: AnsiColorFormat;
```

Comment:
Naming is hard @backwardspy @unseen-ninja 

Are we happy with the keys `normal` / `bright` or should we go for other options like `standard` / `intense` or others. I'm not particularly fussed but pinging pigeon in case he is.

Reply:
normal/bright seems fine to me, feels familiar in the context of ansi